src/Blocks/components/form/assets/utils.js

```diff
@@ -778,6 +778,52 @@ export class Utils {
 		}
 	}
 
+	/**
+	 * Actions to run if api response returns JSON but not formated correct.
+	 *
+	 * This can happen if the API returns JSON but it was malformed for this request like containing debug log or PHP warnings.
+	 *
+	 * @param {mixed} response Api response.
+	 * @param {string} type Function used.
+	 * @param {string} formId Form Id.
+	 *
+	 * @throws Error.
+	 *
+	 * @returns {void}
+	 */
+	formSubmitIsJsonString(response, type, formId) {
+		try {
+			return JSON.parse(response);
+		} catch (e) {
+			if (formId !== null) {
```

Comment:
Please invert this `if`

Reply:
I don' understand?

Reply:
```suggestion
			if (formId === null) {
				throw new Error(`API response returned JSON but it was malformed for this request. Function used: "${type}"`);
			}
```

Reply:
well this is not correct. Error will be thrown no matter if you have formId or not. 

Usage of this is that if formId is used we need to remove loader, global msg etc and throw error if this is a global form we only need to throw and error. So you suggestions are not correct